std_msgs/Header header                  # time stamp
int64   id
float64 completion                      # percentage of completion
float64 attack_accuracy                 # the ratio between pressing button and turning LEDs ON.
float64 press_rate                      # press_counter / total_presses
int64   press_counter                   # num of button presses in the tower